op-node/flags/flags.go

```diff
@@ -19,62 +19,79 @@ import (
 
 const EnvVarPrefix = "OP_NODE"
 
+const (
+	RollupCategory     = "ROLLUP"
+	L1RPCCategory      = "L1 RPC"
+	SequencerCategory  = "SEQUENCER"
+	P2PCategory        = "NETWORKING"
+	OperationsCategory = "LOGGING, METRICS, DEBUGGING, AND API"
+	PlasmaCategory     = "PLASMA (EXPERIMENTAL)"
+)
```

Comment:
we could consider defining them in `op-service/flags` instead, so we could reuse them in other cmds.

Reply:
Yes, would be interesting, but I focused on only adding categories to op-node for now